src/routes/follow.ts

```diff
@@ -0,0 +1,43 @@
+import express from "express";
+import {ensureAuthUser} from "@/middlewares/authentication";
+import {createFollow, deleteFollow} from "@/models/follow";
+
+export const followRouter = express.Router();
+
+followRouter.post(
+  "/follows/:userId",
```

Comment:
RESTの設計に則るとuseridが先のほうが自然そうです！ビュー側のformのアクションURLも修正してもらえると！

```suggestion
  "/:userId/follows",
```